src/transformers/trainer_seq2seq.py

```diff
@@ -28,6 +31,63 @@
 
 
 class Seq2SeqTrainer(Trainer):
+    def __load_generation_config(self, gen_kwargs: Dict[str, Any]) -> Union[GenerationConfig, None]:
```

Comment:
```suggestion
    def _load_generation_config(self, gen_kwargs: Dict[str, Any]) -> Union[GenerationConfig, None]:
```
(nit - we don't use `__` to start function names in the library, unless it is a dunder)

Reply:
We also don't really need an underscore to be honest.